<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushSettings</class>
 <widget class="QWidget" name="BrushSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brush Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QLineEdit" name="brushLabel">
     <property name="placeholderText">
      <string>Brush label</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="GroupedToolButton" name="hardedgeMode">
       <property name="toolTip">
        <string>Hard edged brush</string>
       </property>
       <property name="icon">
        <iconset theme="draw-freehand">
         <normaloff>theme:draw-freehand.svg</normaloff>theme:draw-freehand.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="softedgeMode">
       <property name="toolTip">
        <string>Soft brush</string>
       </property>
       <property name="icon">
        <iconset theme="draw-brush">
         <normaloff>theme:draw-brush.svg</normaloff>theme:draw-brush.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupCenter</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="watercolorMode">
       <property name="toolTip">
        <string>Watercolor</string>
       </property>
       <property name="icon">
        <iconset theme="draw-watercolor">
         <normaloff>theme:draw-watercolor.svg</normaloff>theme:draw-watercolor.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupRight</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0,0">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="4" column="2" colspan="2">
      <widget class="QSlider" name="brushopacity">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" rowspan="2">
      <widget class="QLabel" name="hardnessLabel">
       <property name="text">
        <string>Hardness:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="5">
      <spacer name="colorpickupSeparator">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QSlider" name="brushopacity0">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Opacity:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QSpinBox" name="spinBox_4">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QSlider" name="brushsmudging0">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QSlider" name="brushsize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <spacer name="smudgingSeparator">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QSlider" name="brushhardness0">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QSlider" name="brushsmudging">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1" rowspan="2">
      <widget class="QToolButton" name="hardnessPressure">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="input-tablet">
         <normaloff>theme:input-tablet.svg</normaloff>theme:input-tablet.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="4" rowspan="2">
      <widget class="QSpinBox" name="smudgingBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QSlider" name="brushhardness">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="4" rowspan="2">
      <widget class="QSpinBox" name="hardnessBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Spacing:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="modeLabel">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="5">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0" rowspan="2">
      <widget class="QLabel" name="smudgingLabel">
       <property name="text">
        <string>Smudging:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="5">
      <spacer name="hardnessSeparator">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="2" colspan="2">
      <widget class="QSlider" name="brushspacing">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="2" colspan="2">
      <widget class="QSlider" name="colorpickup">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QSlider" name="brushsize0">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QToolButton" name="opacityPressure">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="input-tablet">
         <normaloff>theme:input-tablet.svg</normaloff>theme:input-tablet.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4" rowspan="2">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QToolButton" name="sizePressure">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="input-tablet">
         <normaloff>theme:input-tablet.svg</normaloff>theme:input-tablet.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="colorpickupLabel">
       <property name="text">
        <string>Color pickup:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QSpinBox" name="colorpickupBox">
       <property name="prefix">
        <string>1/</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="16" column="2" colspan="3">
      <widget class="QCheckBox" name="modeIncremental">
       <property name="text">
        <string>Direct to layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1" rowspan="2">
      <widget class="QToolButton" name="smudgingPressure">
       <property name="toolTip">
        <string>Pressure sensitivity</string>
       </property>
       <property name="icon">
        <iconset theme="input-tablet">
         <normaloff>theme:input-tablet.svg</normaloff>theme:input-tablet.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="5">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/groupedtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>brushsize</tabstop>
  <tabstop>brushhardness</tabstop>
  <tabstop>brushspacing</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>brushsize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushsize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brushopacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushopacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brushhardness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hardnessBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hardnessBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushhardness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brushspacing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushspacing</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizePressure</sender>
   <signal>toggled(bool)</signal>
   <receiver>brushsize0</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacityPressure</sender>
   <signal>toggled(bool)</signal>
   <receiver>brushopacity0</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hardnessPressure</sender>
   <signal>toggled(bool)</signal>
   <receiver>brushhardness0</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smudgingPressure</sender>
   <signal>toggled(bool)</signal>
   <receiver>brushsmudging0</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brushsmudging</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smudgingBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smudgingBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushsmudging</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorpickup</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorpickupBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorpickupBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorpickup</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
